iptables/iptables.go

```diff
@@ -423,12 +423,21 @@ func existsRaw(table Table, chain string, rule ...string) bool {
 	return strings.Contains(string(existingRules), ruleString)
 }
 
+func filterOutput(output []byte, args ...string) []byte {
+	// ignore iptables' message about xtables lock
+	if strings.Contains(string(output), xLockWaitMsg) {
+		logrus.Warnf("Xtables contention detected during: %s", strings.Join(args, " "))
```

Comment:
would be good to still expose the seconds spent waiting for the lock I guess

Reply:
I think the suggestion is to dump the original `output` message as part of the warning i.e. the whole string corresponding to https://git.netfilter.org/iptables/tree/iptables/xshared.c?h=v1.6.0#n261.